### DISCUSSION TITLE
Generating 3D Hilbert curve Fractal using L-systems

### DISCUSSION THREAD

**Discussion by stillpointx** [JacquesLucke/animation_nodes#1805] [Q&A] [answered]
I'm trying to create a connected / joined 3D Hilbert curve.  I want to animated it's growth pattern.  I'm using AN L-Systems but I'm **having issues connecting the line joints fully** and **converting L-systems fully to AN L-systems.**

I'm following / trying to recreate this:
https://math.stackexchange.com/questions/123642/representing-a-3d-hilbert-curve-as-an-l-system

![image](https://user-images.githubusercontent.com/28985314/141868135-b713fea9-29e0-43ef-bce6-4d2bfaa4c448.png)

This is what I have so far notice t**he joints aren't fully connected and I can only get one section**
![image](https://user-images.githubusercontent.com/28985314/141868318-6bc164cd-9b90-4620-900f-1d0d98fe9123.png)

  **[AN_L-systems hilbert.blend.zip](https://github.com/JacquesLucke/animation_nodes/files/7542225/AN_L-systems.hilbert.blend.zip)**

**Comment by OmarEmaraDev** [comment, marked as answer]
You can use the Spline Per Branch from the [Splines From Edges](https://docs.animation-nodes.com/documentation/nodes/spline/splines_from_edges/) node.

**Reply by stillpointx** [reply]
Thanks, I tried that but the image doesn't show up any idea what setting I'm missing?

![image](https://user-images.githubusercontent.com/28985314/142485214-979eb513-9a43-489f-a2cc-145a439683ae.png)


![image](https://user-images.githubusercontent.com/28985314/142484529-91b584f3-922e-43bf-8984-3d98296dc6e3.png)

**Reply by stillpointx** [reply]
I figured it out had to create a new target.001  

Thanks or the help Omar!
